src/services/Store/store/root.reducer.ts

```diff
@@ -59,17 +58,17 @@ export function* importSaga() {
 }
 
 function* importWorker({ payload }: PayloadAction<string>) {
-  const persistable = yield select(exportState);
   try {
-    const json = migrateConfig(JSON.parse(payload));
+    const migrated: LocalStorage = yield call(
+      // @ts-expect-error: bad type choice on call effect
+      migrate,
+      marshallState(JSON.parse(payload)),
```

Comment:
We should probably change our export function to stop requiring `marshallState`. Out of scope for this PR though and will break some exported settings.